doc/user/content/sql/insert.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "INSERT"
+description: "`INSERT` inserts values into a table."
+menu:
+  main:
+    parent: 'sql'
+---
+
+`INSERT` inserts values into a table.
+
+## Conceptual framework
+
+`INSERT` statements inserting data into [tables](../create-table).
```

Comment:
I think mirroring some of the conceptual information for the statement's use cases here would be helpful.